queries.sql

```diff
@@ -16,3 +16,12 @@ select avg(weight_kg) from animals;
 select neutered, count(neutered) as result from animals group by neutered order by result desc limit 1;
 select species, MAX(weight_kg) as max_weight, MIN(weight_kg) as min_weight from animals group by species;
 select species,avg(escape_attempts) from animals where date_of_birth between '1900-01-01' and '2000-12-31' group by species;
+
+
+SELECT name as animal_name, full_name FROM animals A JOIN owners O ON A.owner_id = O.id WHERE full_name = 'Melody Pond';
+SELECT A.name as animal_name, S.name as specie_type FROM animals A JOIN species S ON A.species_id = S.id WHERE S.name = 'Pokemon';
+SELECT full_name, name as animal_name FROM animals A RIGHT JOIN owners O ON A.owner_id = O.id ; 
+SELECT species.name, COUNT(*) FROM animals JOIN species ON animals.species_id = species.id GROUP BY species.name;
+SELECT full_name, A.name FROM animals A JOIN owners O ON A.owner_id = O.id WHERE full_name = 'Jennifer Orwell';
```

Comment:
- [ ] Good job working on this query, but please kindly ensure you list only `all Digimon owned by Jennifer Orwell.`